kolibri/core/assets/src/state/modules/session.js

```diff
@@ -84,6 +84,12 @@ export default {
     isLearnerOnlyImport(state) {
       return !state.full_facility_import;
     },
+    isTouchDevice() {
+      // Check for presence of the touch event in DOM or multi-touch capabilities
+      return (
+        'ontouchstart' in window || navigator.maxTouchPoints > 0 || navigator.msMaxTouchPoints > 0
```

Comment:
Seems like we should probably update this in future too! https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue#L145C36-L145C36

Nice that there is a fairly reliable way to do it now.

Reply:
I agree!